runtime/lua/vim/treesitter.lua

```diff
@@ -108,6 +108,20 @@ function M.create_parser(bufnr, lang, id)
   return self
 end
 
+--- Gets the parser for this bufnr / ft combination.
+--
+-- If needed this will create the parser.
+-- In any case, the provided callbacks will be attached to the buffer.
+--
+-- @param bufnr hte buffer the parser should be tied to
+-- @param ft the filetype of this parser
```

Comment:
This will be annoying, but can you start the sentence after the variable name with an upper-case letter.

`@param ft The filetype...`

This way it is easier to read, IMO